packages/components/tabs/src/Tabs.styles.ts

```diff
@@ -20,7 +20,7 @@ const styles = {
     border: 'none',
     cursor: 'pointer',
     outline: 'none',
-    '&:focus-visible': {
+    '&:focus-visible &:focus': {
```

Comment:
We can't have both a `&:focus-visible` and a `&:focus` here. We should keep only one of them to make it work. 
- with `&:focus` the `boxShadow` will appear always when the item is in the focus state. Either it gets focused by keyboard or by mouse click. 
- with `&:focus-visible` the `box shadow` will appear only when the item is focused by a keyboard.

I would vote for `&:focus-visible` since it is how native `button` for example works. And it looks better 💅